sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/src/Azure.CoreWCF.AzureQueueStorage.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Azure.CoreWCF.AzureQueueStorage</AssemblyName>
+    <PackageId>Azure.CoreWCF.AzureQueueStorage</PackageId>
+    <RootNamespace />
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Storage.Queues" />
+    <PackageReference Include="NuGet.Commands" VersionOverride="6.5.0" />
```

Comment:
These dependencies will need to be approved by the architects and added to the [central package versions](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props).   We generally do not want to include local `VersionOverrides` unless there's a temporary exceptional case.